std/format.d

```diff
@@ -1120,8 +1121,8 @@ struct FormatSpec(Char)
             {
                 if (trailing.ptr[0] == ' ')
                 {
-                    while (!r.empty && std.ascii.isWhite(r.front)) r.popFront();
-                    //r = std.algorithm.find!(not!(std.ascii.isWhite))(r);
```

Comment:
I'll say I'm not opposed to static-import everything if that's the decision. I think the code is cleaner this way, however.